### DISCUSSION TITLE
Line chart -dynamic line colors

### DISCUSSION THREAD

**Discussion by 14837-eng** [apexcharts/apexcharts.js#2675] [Q&A]
How i can change color line when value > n ?

![image](https://user-images.githubusercontent.com/56313646/134788794-f6777fdf-80bd-489d-90aa-005cd0f05ce5.png)

**Comment by junedchhipa** [comment]
Related to https://github.com/apexcharts/apexcharts.js/issues/596
Not implemented yet.

**Reply by 14837-eng** [reply]
(((((
